openpype/modules/base.py

```diff
@@ -341,7 +352,16 @@ def _load_modules():
             fullpath = os.path.join(dirpath, filename)
             basename, ext = os.path.splitext(filename)
 
-            if not os.path.isdir(fullpath) and ext not in (".py", ):
+            if os.path.isdir(fullpath):
+                # Check existence of init fil
+                init_path = os.path.join(fullpath, "__init__.py")
+                if not os.path.exists(init_path):
+                    log.debug((
+                        "Module directory does not contan __init__.py file {}"
+                    ).format(fullpath))
+                    continue
+
+            elif ext not in (".py", ):
                 continue
```

Comment:
Why is this logic duplicated from above? Wouldn't these cases already be 'continued' upon?